hack/test-unit.sh

```diff
@@ -10,6 +10,7 @@ PACKAGES_TO_TEST=(
     "github.com/openshift/console-operator/pkg/console/operator"
     "github.com/openshift/console-operator/pkg/console/starter"
     "github.com/openshift/console-operator/pkg/console/subresource/configmap"
+    "github.com/openshift/console-operator/pkg/console/subresource/consoleserver"
```

Comment:
It appears we have been missing this pkg in our unit tests

Reply:
Not ideal to have to manually add these eh?  Prob should update this script to loop the directory instead, but looks good for now.